style/properties/shorthands/position.mako.rs

```diff
@@ -808,11 +815,37 @@
             Ok(())
         }
     }
+
+    #[cfg(feature = "servo")]
```

Comment:
We need a different impl here because we're feature flagging these properties and gecko isn't.